addons/hw_drivers/tools/helpers.py

```diff
@@ -520,7 +549,90 @@ def get_hostname():
     return socket.gethostname()
 
 
+@cache
+def get_conf_file_path():
+    """
+    Get the path to the odoo.conf file.
+    :return: The path to the odoo.conf file.
+    """
+    system_paths = {
+        'Linux': 'odoo/addons/point_of_sale/tools/posbox/configuration/odoo.conf',
+        'Windows': 'odoo.conf'
+    }
+
+    system = platform.system()
+    path = system_paths.get(system)
+    if path:
+        return path_file(path)
+
+    _logger.error('Unsupported system: %s', system)
+    return None
+
+
+def update_conf(values, section='iot.box'):
+    """
+    Update odoo.conf with the given key and value.
+    :param values: The dictionary of key-value pairs to update the config with.
+    :param section: The section to update the key-value pairs in (Default: iot.box).
+    """
+    conf = get_conf()
+    get_conf.cache_clear()  # Clear the cache to get the updated config
+
+    if not conf.has_section(section):
+        conf.add_section(section)
+
+    for key, value in values.items():
+        conf.set(section, key, value)
+
+    write_file(get_conf_file_path(), conf)
+
+
+@cache
+def get_conf(key=None, section='iot.box'):
+    """
+    Get the value of the given key from odoo.conf, or the full config if no key is provided.
+    :param key: The key to get the value of.
+    :param section: The section to get the key from (Default: iot.box).
+    :return: The value of the key provided or None if it doesn't exist, or full conf object if no key is provided.
+    """
+    conf = configparser.ConfigParser()
+    conf.read(get_conf_file_path())
+
+    return conf.get(section, key, fallback=None) if key else conf  # Return the key's value or the configparser object
+
+
 def disconnect_from_server():
-    """Disconnect the IoT Box from the server"""
-    unlink_file('odoo-remote-server.conf')
+    """Disconnect the IoT Box from the server, clears associated caches"""
     get_odoo_server_url.cache_clear()
+    update_conf({
+        'remote_server': '',
+        'token': '',
+        'db_uuid': '',
+    })
+
+
+# TODO: In the future, remove this method only meant to be used for the migration from the old config files
+# TODO: odoo-db-uuid.conf, odoo-enterprise-code.conf, odoo-remote-server.conf, token to odoo.conf
+def migrate_old_config_files_to_new_config_file():
+    """Migrate old config files to the new odoo.conf"""
+    if not get_conf():
```

Comment:
`get_conf()` without any arguments returns `<configparser.ConfigParser object at 0xefe7f370>` so it looks like we never enter this